### DISCUSSION TITLE
Adding more information to sessions table in database

### DISCUSSION THREAD

**Discussion by ismailnakkar** [laravel/framework#51200] [Ideas]
I have two models that can be authenticated and only the user id is stored in the sessions table. I think it would be useful to add the ability of adding extra field to the session, like user type..

**Comment by TheGodlyLuzer** [comment]
I totally agree with this idea. It would really simplify a lot of things for those applications, using multiple User models.
Maybe just change the user_id to a morph...

**Comment by DGINXREAL** [comment]
+1

**Comment by AideenCarter** [comment]
Good idear!

**Comment by DGINXREAL** [comment]
Maybe for Laravel 12 possible?
